src/utils/graphQL.js

```diff
@@ -0,0 +1,27 @@
+const fetchGraphQL = async (graphQLUrl, url, credentials, app, query) => {
+	try {
+		const fetchUrl = credentials ? url.replace('//', `//${credentials}@`) : url;
+		const response = await fetch(graphQLUrl, {
+			method: 'POST',
+			body: `
+				query{
+					elastic50(host: "${fetchUrl}"){
```

Comment:
Does this only work for elasticsearch 5.0?

Reply:
Okay, so it seems we cannot support 6.x at the moment with [graphql-compose-elasticsearch](https://github.com/graphql-compose/graphql-compose-elasticsearch)?